src/core/pv.jl

```diff
@@ -44,6 +44,7 @@
     can_export_beyond_nem_limit::Bool = off_grid_flag ? false : true,
     can_curtail::Bool = true,
     operating_reserve_required_fraction::Real = off_grid_flag ? 0.25 : 0.0, # if off grid, 25%, else 0%. Applied to each time_step as a % of PV generation.
+    dc_coupled_with_storage::Bool = false
```

Comment:
could you add a short description here?

Reply:
I think we should also add a markdown note on costs when using dc_coupled_with_storage. Users should be aware that our default PV costs include inverts and the O&M costs account for inverter replacements. Similarly, the default BESS per_kw costs don't account for any added costs from it being a hybrid inverter